deploy/server/clusterrole.yaml

```diff
@@ -7,11 +7,11 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: ["create", "get", "list", "watch"]
 - apiGroups: ["policy.open-cluster-management.io"]
-  resources: ["policies", "placementbindings"]
-  verbs: ["create", "get", "list", "watch",]
+  resources: ["policies", "placementbindings", "policies/status"]
```

Comment:
can you move `policies/status` to another separate item? because we never need `create, get, list, watch, delete` permission for `policies/status`, only `update and patch`, right?